### ISSUE TITLE
App Crash at startup.

### ISSUE THREAD

**Issue by Anonymous2716** [SmartPack/PackageManager#101]
I am on Android 9. 

```
java.lang.NullPointerException: Attempt to invoke virtual method 'java.util.Iterator java.util.ArrayList.iterator()' on a null object reference
	at p2.i.c(SourceFile:8)
	at o2.o$d.a(Unknown Source:6)
	at androidx.activity.g.run(SourceFile:53)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
	at java.lang.Thread.run(Thread.java:764)
```

Let me know if you need further information.

**Comment by sunilpaulmathew** [comment]
@Anonymous2716 
Is it crashed just once or always? Thanks

**Comment by Anonymous2716** [comment]
@sunilpaulmathew  always. And right after the loading bar finishes loading. Welcome.

**Comment by Anonymous2716** [comment]
Seems to be fixed on v7.0
